### ISSUE TITLE
How to start

### ISSUE THREAD

**Issue by batata004** [PowerShell/Win32-OpenSSH#1396]
Hi,

After almost 40min I am still trying to figure it out how to START the SFTP server on my Windows (which exe file should I execute and which command line options cause I cant find) to allow incoming connections! Can someone please point me in the right link so I can check how to start the SFTP/SSH server to allow my windows to received external files from other SSH machines running Linux?

**Comment by manojampalam** [comment]
Its no different from how it works on Unix. 

There isn't a SFTP server in specific. SFTP is implemented a subsystem in OpenSSH and is enabled by default in Windows. You just start sshd service and connect using sftp client. See if this helps:
https://github.com/PowerShell/Win32-OpenSSH/wiki/sftp.exe-examples

**Comment by 4evermaat** [comment]
I'm not able to connect using FileZilla (`win2012 R2 x64, openSSH 8.0.0.0`)

```
Status: Connecting to xxx.xxx.xxx.xxx...
Response: fzSftp started, protocol_version=8
Command: open "user@xxx.xxx.xxx.xxx" 22
Error: Connection timed out after 20 seconds of inactivity
Error: Could not connect to server
```

But on a different server (win 2008) with a previous version of OpenSSH, I followed the install instructions and it all worked on the first attempt. 

Using the same FileZilla client, I can still connect to win2008 server, but this win2012 will not.

I've spent several hours trying on my own to see what could possibly be wrong.

**Comment by maertendMSFT** [comment]
Can you provide the server side logs?

**Comment by 4evermaat** [comment]
> Can you provide the server side logs?

If you are talking to me, I was able to get OpenSSH to work.   It had something to do with the way my VPS was setup.   When my permanent IP address was assigned, i was able to do it.   But when I had a different IP address setup, the NAT server setup was different.  I was able to access "regular" internet via VPS without any issues.  But  I was unable to get the OpenSSH server to receive requests.

There were some trouble tickets with VPS provider....i'd have to go back and look at them.
